lib/miq_automation_engine/service_models/miq_ae_service_compliance_detail.rb

```diff
@@ -0,0 +1,7 @@
+module MiqAeMethodService
+  class MiqAeServiceComplianceDetail < MiqAeServiceModelBase
+    expose :compliance, :association => true
+    expose :condition, :association => true
```

Comment:
@jockey10 There is no service model for the `Condition` model.  You either need to add it or drop this association.